app/experimenter/visualization/api/v3/views.py

```diff
@@ -16,23 +20,60 @@ def load_data_from_gcs(filename):
     )
 
 
-@api_view()
-def analysis_results_view(request, slug):
-    experiment = get_object_or_404(NimbusExperiment.objects.filter(slug=slug))
-    experiment_data = {}
+def get_results_metrics_map(features):
+    # A mapping of metric label to relevant statistic. This is
+    # used to see which statistic will be used for each metric.
+    RESULTS_METRICS_MAP = {
+        "retained": "binomial",
+        "search_count": "mean",
+        "identity": "count",
+    }
+    for feature in features:
+        feature_metric_id = f"{feature}_ever_used"
+        RESULTS_METRICS_MAP[feature_metric_id] = "binomial"
 
-    recipe_slug = experiment.slug.replace("-", "_")
+    return RESULTS_METRICS_MAP
+
+
+def append_population_percentages(data):
+    total_population = 0
+    branches = {}
+    for row in data:
+        if row["metric"] == USER_COUNTS_KEY:
+            total_population += row["point"]
+            branches[row["branch"]] = row["point"]
 
-    daily_data_filename = f"""statistics_{recipe_slug}_daily.json"""
-    weekly_data_filename = f"""statistics_{recipe_slug}_weekly.json"""
+    for branch_name, branch_user_count in sorted(branches.items()):
+        data.append(
+            {
+                "metric": "identity",
+                "statistic": "percentage",
+                "branch": branch_name,
+                "point": round(branch_user_count / total_population * 100),
+            }
+        )
 
-    daily_data = load_data_from_gcs(daily_data_filename)
-    weekly_data = load_data_from_gcs(weekly_data_filename)
 
-    if daily_data:
-        experiment_data["daily"] = daily_data
+def get_data(slug, window):
+    data = {}
```

Comment:
Isn't this a no-op?

Reply:
Yea I must have been testing something and forgot to remove it. Thanks